src/inquirer/render/console/_checkbox.py

```diff
@@ -89,6 +91,11 @@ def process_input(self, pressed):
         elif pressed == key.RIGHT:
             if self.current not in self.selection:
                 self.selection.append(self.current)
+        elif pressed == key.CTRL_A:
```

Comment:
You added new branches to the code without adding tests for them. So this change request is reducing our test coverage. Please update the tests.